android/app/src/main/java/org/fossasia/openevent/core/auth/profile/EditProfileActivity.java

```diff
@@ -88,6 +92,8 @@ protected void onCreate(Bundle savedInstanceState) {
             Views.hideKeyboard(this, this.getCurrentFocus());
             saveChanges();
         });
+
+        ActivityCompat.requestPermissions(EditProfileActivity.this, READ_STORAGE, REQUEST_CODE);
```

Comment:
Where is the permission response handled?

Reply:
@iamareebjamal Why do we need to handle the permission response ? Like we just need the app to allow access to external storage right ?

Reply:
What if user declined? The app will crash as before. Then what's the point of this PR?